psp/data/data_sources/nwp.py

```diff
@@ -1,25 +1,25 @@
 import pathlib
 import pickle
-from typing import Tuple, TypeVar
+from typing import TypeVar
 
 # This import registers a codec.
 import ocf_blosc2  # noqa
-import pyproj
 import xarray as xr
 
+from psp.gis import CoordinateTransformer
 from psp.typings import Timestamp
 from psp.utils.dates import to_pydatetime
 from psp.utils.hashing import naive_hash
 
-_transformer = pyproj.Transformer.from_crs(4326, 27700)
-
-
-def _to_osgb(points: list[Tuple[float, float]]) -> list:
-    return list(_transformer.itransform(points))
-
-
 T = TypeVar("T", bound=xr.Dataset | xr.DataArray)
 
+_X = "x"
+_Y = "y"
+_TIME = "time"
+_STEP = "step"
+_VARIABLE = "variable"
```

Comment:
for our case the dimension is "variables"?

Reply:
I think the default should be "variable" singular!